AsyncDisplayKit/Private/ASDisplayNode+UIViewBridge.mm

```diff
@@ -174,6 +174,7 @@ - (void)setNeedsDisplay
 - (void)setNeedsLayout
 {
   _bridge_prologue;
+  [self __setNeedsLayout];
   _messageToViewOrLayer(setNeedsLayout);
```

Comment:
I missed this the first time around.  Now I understand this will still call CA / UIKit.  Whew, we're good to land.